Library/Formula/aws-sdk-cpp.rb

```diff
@@ -0,0 +1,70 @@
+class AwsSdkCpp < Formula
+  desc "AWS SDK for C++"
+  homepage "https://github.com/aws/aws-sdk-cpp"
+  url "https://github.com/aws/aws-sdk-cpp/archive/0.9.6.tar.gz"
+  sha256 "cf2dad62e50c5c5cff88a4ad959af3aa402aa7e8aa1692bdf807052c0fe4aeb9"
+  head "https://github.com/aws/aws-sdk-cpp.git"
+
+  ALL_CLIENTS = [
```

Comment:
@dunn What if we did something like:

`brew install aws-sdk-cpp` will install all of the clients
`brew install aws-sdk-cpp --without-compute --without-storage` will install all clients *except* clients from the "compute" and "storage" categories

This isn't really ideal for any scenario I can see someone wanting, but I guess it conforms more to your homebrew style.